samples/blogger/blogger.js

```diff
@@ -13,18 +13,16 @@
 
 'use strict';
 
-var google = require('../../lib/googleapis');
-var blogger = google.blogger('v3');
-var util = require('util');
+const google = require('googleapis');
+const blogger = google.blogger('v3');
+const config = require('../config.json');
```

Comment:
Normally we read api key from the environment variable, which makes it easier to test the samples code in the continuous integration (which we don't do now anyway). If it's too much change, I'm ok to leave it in the file.

Reply:
It's not a huge deal.  Typically if I'm going to do that, I like to use `nconf` so I can support both file based and env based methods.  I'll take a shot at that tonight.